src/ESPEasyWifi.ino

```diff
@@ -105,11 +105,10 @@ void processGotIP() {
   }
 
   #ifdef FEATURE_MDNS
-
+    bool mdns_started = MDNS.begin(WifiGetHostname().c_str(), WiFi.localIP());
     if (loglevelActiveFor(LOG_LEVEL_INFO)) {
       String log = F("WIFI : ");
-      if (MDNS.begin(WifiGetHostname().c_str(), WiFi.localIP())) {
```

Comment:
related to #2048 ??

Reply:
No is not related to that issue, but indeed looks like a bug to me.